### DISCUSSION TITLE
Sharpening with gatsby-plugin-sharp

### DISCUSSION THREAD

**Discussion by BastienFoucher** [gatsbyjs/gatsby#28731] [Help]
Hello from France :)

First post here : I am working on a image-heavy portfolio website. I use gatsby-image a lot and it works very well, thank you for all the work !

I am just a little disapointed of the downsized images, they are not very sharp. I tried the MozJPEG encoding option, without significant improvement on the sharpness of the images. One workaround is to prevent the browser to display resized pictures. 

So I'd be glad if we could through Gatsby Plugin Sharp call the sharpen API of the sharp library. Do you know if it is possible ? I didn't find how to do it.

Thank you !

Bastien

**Comment by KyleAMathews** [comment]
Hello from America :-)

You probably want to set the `quality` option. By default it's set to minimize file size while keeping quality adequate. You might want to set it higher like 80 or 90.

**Reply by BastienFoucher** [reply]
Hello Kyle, thank you ! :) 

No it is not the quality settings that I am looking for. What I'd love is having an access to the "sharpen" sharp image operation : https://sharp.pixelplumbing.com/api-operation#sharpen

There is an access to the rotate operation in gatsby-plugin-sharp, so it might be possible to access the sharpen operation as well.

That would be a great improvement for photographic websistes where picture quality and sharpness is important.

Like I really need it, I am going to try to make the change in the plugin. If my changes works, I will see how to submit my change to the plugin project.
